projects/laji-ui/src/lib/sidebar/sidebar.component.html

```diff
@@ -53,7 +53,7 @@
 </ng-template>
 
 <ng-template #closeBtn>
-  <div class="close-wrapper">
+  <div class="close-wrapper" [ngClass]="{'mobile': mobile}">
```

Comment:
Seems unnecessary to add the class here since the sidebar wrapper already has that class. The css selector can be `.mobile .close-wrapper`

Reply:
It actually does need it to trigger the subclass of close-wrapper which overrides the button position in close-wrapper itself.